doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -865,6 +865,9 @@ Quitting and debugging
    Executes :n:`@sentence`, redirecting its
    output to the file ":n:`@string`.out".
 
+   The directory in which the file is created (or overriden) can be set
+   with the command line option `-output-directory` (see
+   :ref:`command-line-options`).
```

Comment:
See my comment above in universe-polymorphism.rst.

Reply:
Modified too, thanks.